packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1016,15 +1017,18 @@ abstract class CustomClipper<T> {
   /// Creates a custom clipper.
   ///
   /// The clipper will update its clip whenever [reclip] notifies its listeners.
-  const CustomClipper({ Listenable reclip }) : _reclip = reclip;
+  const CustomClipper({ Listenable reclip, this.clip = Clip.antiAlias }) : _reclip = reclip;
 
   final Listenable _reclip;
 
   /// Returns a description of the clip given that the render object being
   /// clipped is of the given size.
-  T getClip(Size size);
+  T getClipShape(Size size);
```

Comment:
Renaming this is a breaking change.

Reply:
The old name is unfortunate, for sure, but I'm not sure it's unfortunate enough to warrant the cost of breaking customers.

Reply:
There's a bunch of privates that you had to rename to avoid conflicting with `clip` too. Maybe we should rename `clip` to `clipBehavior` or `clipRegime` or some such?

Reply:
In that case, maybe we shall just call it `clipMode` to be consistent with flutter/engine?